src/stage0.txt

```diff
@@ -12,15 +12,15 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.(x+1).0` for Cargo where they were released on `date`.
 
-date: 2020-10-16
-rustc: beta
-cargo: beta
+date: 2020-11-16
+rustc: 1.48.0
+cargo: 1.48.0
```

Comment:
Shouldn't this be 0.49.0?

Reply:
https://github.com/rust-lang/rust/pull/77336 changed the dist code to always use the Rust version number for stable tarballs, even for tools like Cargo. Opened https://github.com/rust-lang/rust/pull/79133 to remove the `cargo` field from `src/stage0.txt` :)